### DISCUSSION TITLE
Change sam vit l model，don't use default model

### DISCUSSION THREAD

**Discussion by guoguolord** [wkentaro/labelme#1268] [Ideas / Feature requests]
I want to change another sam models ,don't use default model. but I don't know how to change, even don't know whether support or

**Comment by vietanhdev** [comment]
@guoguolord In the meantime, you may want to try AnyLabeling - a fork of LabelMe with Segment Anything, YOLO, and OCR labeling support.
![image](https://user-images.githubusercontent.com/18329471/236682109-00cd9577-f613-4d34-941a-e411a4e15ae4.png)
- Repository: https://github.com/vietanhdev/anylabeling
- Documentation: https://anylabeling.com/docs.

**Comment by wkentaro** [comment]
@guoguolord now you can choose 3 sam models from a dropdown: https://github.com/wkentaro/labelme/pull/1262#issuecomment-1542288650
